Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json

```diff
@@ -129,6 +129,7 @@
       "ny_ic_day" : true,
       "ny_ic_dest_marker" : true,
       "ny_ic_direct_call" : true,
+      "ny_ic_dismiss" : true,
```

Comment:
can you add the image for this

Reply:
The image has already been added for this in asset store and android as well as in IOS and PR was merged only this JSON had to be pushed

Reply:
okay

Reply:
okay